src/index.ts

```diff
@@ -18,18 +18,24 @@ interface AuthenticateResult {
 }
 
 export interface CreateJWSOptions {
+  did?: string
   protected?: Record<string, any>
+  linkedBlock?: string
 }
 
 interface CreateJWSParams extends CreateJWSOptions {
-  did: string
-  payload: DagJson
+  payload: string | any
```

Comment:
Maybe we can use `Record<string, any>` rather than `any` if the payload is expected to be an object?

Reply:
Payload can really be anything that is valid JSON.

Reply:
👍 we could just use `any` then as `string` is included in `any`

Reply:
Sure!